aurora/facit.py

```diff
@@ -0,0 +1,1461 @@
+        poloidal current flux function [:math:`T*m`]
+        required if rotation_model == 1 and full_geom
+        default is None, which makes it R0*B0
+    dpsidx : 1D array (nr), optional
+        radial coordinate transformation from flux psi to rho [:math:`T*m^2`]
+        required if rotation_model == 1 and full_geom
+        default is None
+    fsaout : bool, optional
+        if True, return FSA transport coefficients
+        only relevant if rotation_model == 2
+        default is True
+    full_geom : bool, optional
+        only relevant if rotation_model == 1
+        if True, use full flux surface shaped geometry in the calculation of
+        the asymmetries, which means an iterative solution. This would be the
+        most computationally-expensive setting of FACIT.
+        if False, use circular geometry with large aspect ratio, such that
+        the calculation of the asymmetries can be completely analytical
+        default is False
+    nat_asym : bool, optional
+        if True, consider friction-induced "natural" poloidal asymmetry
+        only relevant if pol_asym
+        default is True
+    nth : int, optional
+        number of poloidal grid points [-]
+        if RV, BV, ZV are given, it is the size of axis 1,
+        else default is 20
+    regulopt : list (4), optional
+        parameters for the convergence of the iterative calculation of the
+        asymmetries
+        only relevant if rotation_model = 1 and full_geom
+        default is [1e-2,0.5,1.e-5,1e2]
+    fH : float, optional
+        resonant hydrogen minority fraction [-]
+        for ICRH-induced asymmetries
+        only relevant if pol_asym
+        default is 0.0
+    bC : float, optional
+        Bres/B0 [-], where Bres is the field where the ICRH frequency matches the 
+        fundamental cyclotron resonance of the minority ion: 2*pi*f = Zi*e*Bres/mi
+        for ICRH-induced asymmetries
+        only relevant if pol_asym
+        default is 1.0
+    sigH : float, optional
+        std. dev. for radial exponential decay of temperature anisotropy
+        for ICRH-induced asymmetries
+        only relevant if pol_asym
+        default is 1.0
+    TperpTpar_axis : float, optional
+        main ion temperature anisotropy at magnetic axis [-]
+        for ICRH-induced asymmetries
+        only relevant if pol_asym
+        default is 1.0
+        
+    Returns
+    -------
+    Dz_* : 1D array (nr)
+        diffusion coefficient for each flux component [:math:`m^2/s`]
+        * = PS, BP or CL (for Pfirsch-Schlüter, Banana-Plateau and Classical)
+    Kz_* : 1D array (nr)
+        coefficient of the main ion density gradient for each flux component [:math:`m^2/s`]
+    Hz_* : 1D array (nr)
+        coefficient of the main ion temperature gradient for each flux component [:math:`m^2/s`]
+    Vrz_* : 1D array (nr)
+        radial flux per particle (Flux_z/nz)_* for each flux component [:math:`m/s`]
+    Vrz_* : 1D array (nr)
```

Comment:
Vrz_ is here twice

Reply:
I suspect that the second one should be deleted; there doesn't seem to be a variable like Vconv_* fr each flux component. @DanielFajardoJ could you please confirm and make the correction?

Reply:
Also, I think that we way want to substitute the header "Returns" with "Returns (as attributes)". The __init__() method here does not actually use a return statement.

Reply:
The second Vrz_* was wrong, it should have been Vz_* for the convection of each PS, BP, CL flux component which were the attribute names. I now changed these names to Vconv_* for consistency and updated the header (also the "Returns (as attributes)").